CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 * The AllergyIntoleranceMapper has been enhanced to support the redaction fix. If an Allergy Intolerance record includes a confidentialityCode, the meta.security field of the corresponding FHIR resource will now be appropriately populated.
+* **BREAKING CHANGE**: The GP2GP Translator Service will now exit and throw a DataAccessException with a message of
```

Comment:
I would argue it isn't a breaking change, I don't think any end users want to look at this.

Reply:
Fixed